src/llv8.h

```diff
@@ -403,6 +403,8 @@ class LLV8 {
 
   void Load(lldb::SBTarget target);
 
+  constants::Types types;
```

Comment:
For that I have to add forward declarations for the class FindJSObjectsVisitor and the GetTypeName function (and it's parameters, one of which is another inner class) in llv8.h.
If GetTypeName moved though it would be simpler and we'd just need the forward declaration for FindJSObjectsVisitor.